Earthfile

```diff
@@ -350,7 +350,7 @@ ci-artifacts:
 
 # ci-codeql-setup sets up CodeQL for the ci-codeql target.
 ci-codeql-setup:
-  ARG CODEQL_VERSION=v2.17.3
+  ARG CODEQL_VERSION=v2.25.6
```

Comment:
Interesting - looks like there's two forms of tag in this repo. We want the ones prefixed with `codeql-bundle`, of which 2.17.3 is actually the latest.

https://github.com/github/codeql-action/tags